packages/hagrid/hagrid/deps.py

```diff
@@ -500,16 +543,23 @@ def check_docker_service_status(animated: bool = True) -> None:
     """
 
     if not animated:
-        if not docker_running():
-            raise MissingDependency("❌ Docker service is not running.")
-
+        docker_installed, msg = docker_running()
+        user_allowed, permission_msg = allowed_to_run_docker()
     else:
         console = Console()
         # putting \t at the end seems to prevent weird chars getting outputted
         # during animations in the juypter notebook
         with console.status("[bold blue]Checking for Docker Service[/bold blue]\t"):
-            if not docker_running():
-                raise MissingDependency("❌ Docker service is not running.")
+            docker_installed, msg = docker_running()
+            user_allowed, permission_msg = allowed_to_run_docker()
```

Comment:
here we can swap the order of these method calls. 
So first call `allowed_to_run_docker()` and then call `docker_running` method.